pkg/piecestore/psserver/psdb/testdata/sqlite.v4.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE `ttl` (`id` BLOB UNIQUE, `created` INT(10), `expires` INT(10), `size` INT(10));
+CREATE TABLE `bandwidth_agreements` (`satellite` BLOB, `agreement` BLOB, `signature` BLOB, `uplink` BLOB, `serial_num` BLOB, `total` INT(10), `max_size` INT(10), `created_utc_sec`	INT(10), `expiration_utc_sec` INT(10), `action` INT(10), `daystart_utc_sec` INT(10), `status` INT(10) DEFAULT 0);
+CREATE INDEX idx_ttl_expires ON ttl (expires);
```

Comment:
This file need to have the same `v` number like migration number so it shoulde `v5`

Reply:
oh sorry I didn't saw next file. if this is the same file like v5 then it needs to be removed.

Reply:
yes it is same file.. I thought it needs to have a sequence of file. .. I will remove it.

Reply:
done

Reply:
file is still in PR